packages/backend/src/open_payments/payment/incoming_remote/service.ts

```diff
@@ -120,10 +120,20 @@ async function getGrant(
 
   if (existingGrant) {
```

Comment:
I also noticed: since `grant.accessToken` is now always defined, we can remove the `if (!grantOrError.accessToken) {` check on L67 in this service